keps/sig-instrumentation/2831-kubelet-tracing/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: Kubelet OpenTelemetry Tracing
+kep-number: 2831
+authors:
+  - "@husky-parul"
+  - "@somalley"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-architecture
+  - sig-node
+status: provisional
```

Comment:
this needs to be implementable if we plan to implement it in 1.24

Reply:
updated